src/main/scala/is/hail/methods/KinshipMatrix.scala

```diff
@@ -122,9 +124,9 @@ case class KinshipMatrix(val hc: HailContext, val matrix: IndexedRowMatrix, val
   def exportIdFile(idFile: String) {
     hc.sc.hadoopConfiguration.writeTextFile(idFile) { s =>
       for (id <- sampleIds) {
-        s.write(id)
+        s.write(id.asInstanceOf[String])
```

Comment:
seems unsafe. Goes along with my comments about moving the schema checks to scala.